<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewWidget</class>
 <widget class="QWidget" name="LogViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="lblStartDate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Début</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateStartDate">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string notr="true">dd-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
       <property name="date">
        <date>
         <year>1999</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblEndDate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEndDate">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string notr="true">dd-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
       <property name="date">
        <date>
         <year>1999</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblLevel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLevel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFilter">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Filtrer</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/TeraClient.qrc">
         <normaloff>:/icons/filter.png</normaloff>:/icons/filter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Rafraichir</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/TeraClient.qrc">
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblWarning">
     <property name="text">
      <string>Aucun résultat n'est disponible.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableLogs">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Heure</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameNav">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblEntriesCount">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">xxx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblEntries">
        <property name="text">
         <string>événements</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnPrevPage">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string notr="true">&lt;&lt;&lt;</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/TeraClient.qrc">
          <normaloff>:/controls/branch_left.png</normaloff>:/controls/branch_left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinPage">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTotalPages">
        <property name="text">
         <string notr="true">/ 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnNextPage">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string notr="true">&gt;&gt;&gt;</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/TeraClient.qrc">
          <normaloff>:/controls/branch_closed.png</normaloff>:/controls/branch_closed.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/TeraClient.qrc"/>
 </resources>
 <connections/>
</ui>
